### PR TITLE
Updated temperature-sensor.md

### PR DESCRIPTION
**Pull request by AryaDeepanshu**
## Types of changes:
- [x] Correct a spelling
- [x] Add new information
- [x] Change information

## Checklist:
- [x] I have proofread the changes. (Spellings, grammar, etc)
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: docs/components/sensors/temperature-sensor.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+[ Getter Only ]  
+Type: Number   
+![image](url)   
+
+Return the temperature in degree Celsius.
+## Events  
+### Temperature Changed  
+[ Number `Tempereature` ]   
+![image](url)  
+Event that fires when temperature is changed.
```
Comment (Vishwas-Adiga) [inline docs/components/sensors/temperature-sensor.md]: Please use standard sentence structures, and a table for event parameters. Use screenshot.md as reference

File: docs/components/sensors/temperature-sensor.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
-# Temperature Sensor
 
-{!includes/sip.md!}
+# Temperature
```
[the remark this answers is not in the document]
Comment (AryaDeepanshu) [reply docs/components/sensors/temperature-sensor.md]: done

File: docs/components/sensors/temperature-sensor.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
...
+
+## Overview
+  
+A non-visible Component that measures the temperature of air surrounding the device in unit of Degree Celsius.
```
[the remark this answers is not in the document]
Comment (AryaDeepanshu) [reply docs/components/sensors/temperature-sensor.md]: fixed

File: docs/components/sensors/temperature-sensor.md [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+
+  
+## Properties
+
+  
+
+### Enabled 
+[ Getter + Setter ] 
+Type: Boolean
```
[the remark this answers is not in the document]
Comment (AryaDeepanshu) [reply docs/components/sensors/temperature-sensor.md]: fixed

File: docs/components/sensors/temperature-sensor.md [hunk 5]
```diff
@@ -1,3 +1,40 @@
...
+
+  
+
+### Enabled 
+[ Getter + Setter ] 
+Type: Boolean    
+
+![image](url) 
+
+If checked the sensor will be enabled
```
[the remark this answers is not in the document]
Comment (AryaDeepanshu) [reply docs/components/sensors/temperature-sensor.md]: fixed

File: docs/components/sensors/temperature-sensor.md (hunk 1 above)
Comment (AryaDeepanshu) [reply docs/components/sensors/temperature-sensor.md]: fixed

File: docs/components/sensors/temperature-sensor.md [hunk 6]
```diff
@@ -1,3 +1,40 @@
...
+
+### Temperature 
+[ Getter Only ]  
+Type: Number   
+![image](url)   
+
+Return the temperature in degree Celsius.
+## Events  
+### Temperature Changed  
+[ Number `Tempereature` ]
```
[the remark this answers is not in the document]
Comment (AryaDeepanshu) [reply docs/components/sensors/temperature-sensor.md]: fixed, Grammarly shows no spelling mistake

**Pull request closed without merging**